scripts/xtensa-build-all.sh

```diff
@@ -35,6 +38,8 @@ usage: xtensa-build.sh [options] platform(s)
        -v Verbose Makefile log
        -j n Set number of make build jobs. Jobs=#cores when no flag. \
 Infinte when not specified.
+	-m MEU path
+	-s openssl path
```

Comment:
Assuming I understood the new code correctly:
```
-m path to MEU tool. Switches signing step to use MEU instead of rimage.
-s path to openssl tool. Requires -m and required by -m.
```